site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Last Week in ManageIQ: Exploring Budapest while the others are working"
+author: skateman
+date: 2017-10-02
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello everyone, my name is [Dávid Halász](https://github.com/skateman) and I'd like to guide you through the changes we made in the last week.
+
+## Featured
+
+During the weekend [Roman](https://github.com/romanblanco), [Julian](https://github.com/juliancheal) and myself were attending at [EuRuKo 2017](https://www.euruko2017.org/) in Budapest. It is a beautiful city where you can among others eat [halászlé](https://hu.wikipedia.org/wiki/Hal%C3%A1szl%C3%A9) (not named after me) and see the [Halászbástya](https://hu.wikipedia.org/wiki/Hal%C3%A1szb%C3%A1stya) (also not named after me).
```

Comment:
wondering if it's better to use the English wikipedia links? Up to you...

Reply:
@cybette they're literally translated so won't make that much sense :disappointed:

Reply:
oh my bad, it's there, changing it